### PR TITLE
Apply Prettier to markdown files

### PR DESCRIPTION
**Pull request by mrtnzlml**
Currently, we are checking only JS files via Eslint. This change adds additional check to markdown files as well.

### CODE REVIEW

File: src/babel-preset-adeira/CHANGELOG.md [hunk 1]
```diff
@@ -55,12 +55,12 @@ Changelog before our fork:
 - Added support for dynamic `import()` syntax. This fits well with `@adeira/relay` version 1.0+ and allows you to use `@match`/`@module` directives for data-driven code splitting.
 - Breaking: Relay plugin is no longer part of this preset. It was moved to `@adeira/relay`. Please, edit your Babel configuration files (example for Next.js applications):
 
-```js
+\`\`\`js
```
Comment (michalsanger) [inline src/babel-preset-adeira/CHANGELOG.md:58]: This does not look OK.

Comment (mrtnzlml) [reply src/babel-preset-adeira/CHANGELOG.md:58]: How should we do it differently? We are nesting there one code block into another, see: https://github.com/adeira/universe/blob/5d3733e6ea9c04c32e2d434d394b0081bdb0afe3/src/babel-preset-adeira/CHANGELOG.md

Comment (michalsanger) [reply src/babel-preset-adeira/CHANGELOG.md:58]: Options I see:

1. close the block before the one with js example
2. remove the the triple backtick
3. don't format the whole "Changelog before our fork" as a code block

Comment (mrtnzlml) [reply src/babel-preset-adeira/CHANGELOG.md]: None of the aforementioned options are what I wanted to do. But it's fixed now, please have a look.

**Pull request merged**